src/screens/Game/GamePicture/index.tsx

```diff
@@ -1,21 +1,52 @@
 import React from 'react'
-
 import './index.css'
-
+import EpicLogo from '../../../assets/epic-logo.svg'
 type Props = {
   art_square: string
+  store: string
 }
 
-function GamePicture({ art_square }: Props) {
+function GamePicture({ art_square, store }: Props) {
   return (
     <div className="gamePicture">
       <img
         alt="cover-art"
-        src={`${art_square}?h=800&resize=1&w=600`}
+        src={
+          store == 'legendary'
+            ? `${art_square}?h=800&resize=1&w=600`
+            : art_square
+        }
         className="gameImg"
       />
+      <div className="store-icon">
+        {store == 'legendary' ? <img src={EpicLogo} alt="" /> : GOGLogo()}
+      </div>
     </div>
   )
 }
 
 export default GamePicture
+
+function GOGLogo() {
```

Comment:
I believe you can store this on an SVG file and import it like we are doing with other SVG files.
we import them as react components so we can pass properties to it and use as normal component.
I believe for both icons could be this way.